test/simulation/README.md

```diff
@@ -8,38 +8,63 @@ The simulation tests’ only dependencies are [Docker](https://docs.docker.com/i
 
 ## Usage (via `xud` npm scripts)
 
-Run the tests after building all images:
+#### Run the tests after building all images:
 
 ```bash
 $ npm run test:sim:build
 $ npm run test:sim:run
 ```
 
-Whenever test code is being refactored, one must build the `test` image before re-running the tests so that changes could take effect:
+#### Specific builds
+
+For taking into effect changes in test code, one must build the `test` image before re-running the tests:
 
 ```bash
 $ npm run test:sim:build:test
 $ npm run test:sim:run
 ```
---- 
 
-The `test:sim:run` script mentioned above runs only the Integration tests suite.
+For taking into effect changes in `xud` (non-test, production code), one must build the `xud` image before re-running the tests, in addition to cleaning its volume:
```

Comment:
Here also I'd say "When making changes to non-test `xud` code, one must clean and build the `xud` image before rerunning the tests."

Reply:
The "clean" is being referred to the volume, not the image (which always get overriden, unlike the volume). This is why it's distinct.

Reply:
I see, how about "one must build the `xud` image and clean its volume before rerunning..." ?